wowutil.py

```diff
@@ -65,6 +72,30 @@ def install_db_extensions(conn):
     conn.commit()
 
 
+def create_and_populate_oca_tables(conn, is_testing: bool = False):
+    import ocaevictions.table
+
+    config = ocaevictions.table.OcaConfig(
+        oca_table_names=OCA_TABLES,
+        sql_dir=WOW_SQL_DIR,
+        data_dir=NYCDB_DATA_DIR,
+        test_dir=TEST_DATA_DIR,
+        oca_db_url=os.environ.get("OCA_DATABASE_URL", None),
+        oca_ssh_host=os.environ.get("OCA_SSH_HOST", None),
+        oca_ssh_user=os.environ.get("OCA_SSH_USER", None),
+        oca_ssh_pkey=os.environ.get("OCA_SSH_PKEY", None),
+        is_testing=is_testing,
+    )
+
+    with conn.cursor() as wow_cur:
+        slack.sendmsg("Rebuilding OCA evictions tables...")
+        ocaevictions.table.create_oca_tables(wow_cur, config)
+        ocaevictions.table.populate_oca_tables(wow_cur, config)
```

Comment:
could handle errors to just warn/error in rollbar but continue with the rest of the wow dataset job